ui/packages/consul-ui/app/models/topology.js

```diff
@@ -33,14 +31,19 @@ export default class Topology extends Model {
     return undefinedDownstream;
   }
 
-  @computed('FilteredByACL', 'DefaultAllow', 'WildcardIntention', 'notDefinedIntention')
```

Comment:
I removed this code because it was not efficient to show or not the collapsible banner. Soon it will be impossible to figure out all the "notice combos" that will exist.

Reply:
Also, we did find out with BE folks that default allow and wildcard intentions can exist at the same time.